.github/workflows/chromatic.yml

```diff
@@ -0,0 +1,36 @@
+name: Visual regression testing
+
+on:
+  pull_request:
+    branches:
+      - master
+    types:
+      - labeled
+  workflow_dispatch:
+
+jobs:
+  build-pr:
+    environment: pull_request_unsafe
+    if: (github.ref == 'refs/heads/master') || (github.event.label.name == 'need visual approval')
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Use Node.js 14
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14
+          check-latest: true
+          registry-url: "https://registry.npmjs.org/"
+          scope: "@talend"
+          cache: "yarn"
+
+      - name: Install dependencies
+        run: yarn
```

Comment:
```suggestion
        run: |
          cd packages/design-system
          yarn
```

Reply:
no you are in a mono repository so this do not change anything. The lockfile is at the root